catalyst-gateway/bin/src/follower.rs

```diff
@@ -200,8 +205,12 @@ async fn init_follower(
                         },
                     };
 
-                    // Parse block
+                    if !valid_era(block.era()) {
```

Comment:
I think we should still keep the `cardano_slot_index` up to date for every era, just not do any other processing of the block.  So this would move to 255 and we can invert the condition to `if valid_era`.

We should also update the last_updated table for every era, otherwise we will have a potential that a failure syncing during byron always forces us to reset back to genesis, we can't keep going unless we get past byron.

Reply:
Ok cool,

I changed it to "process" block stuff such as txs, registrations, rewards etc when it's a valid era but continue valid+invalid for **cardano_slot_index** and update metadata (syncing purposes).